operators/src/engine/query.rs

```diff
@@ -8,10 +8,22 @@ pub struct QueryRectangle {
     pub spatial_resolution: SpatialResolution,
 }
 
-/// A collection of meta data for processing a query
-#[derive(Copy, Clone, Debug)]
-pub struct QueryContext {
-    // TODO: resolution, profiler, user session, ...
-    // TODO: determine chunk size globally or dynamically from workload? Or global Engine Manager instance that gives that info
+pub trait QueryContext: Send + Sync {
+    fn chunk_byte_size(&self) -> usize;
+}
+
+pub struct MockQueryContext {
```

Comment:
>jdroenner commented on 20.01.2021
why not call it something like SimpleQueryContext or something likt this?

because it should only ever be used in tests

Reply:
The name is fine for me :shrug:

Reply:
Can you think of any concrete usages other than testing for this? If there are any we can rename it, otherwise we should leave it like it is and change it if there is ever the need.